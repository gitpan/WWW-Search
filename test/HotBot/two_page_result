http://www.search.hotbot.com/hResult.html?SM=SC&MT=%2B%22Martin+Thurn%22+%2BSWB&DE=2&DC=84&
http://www.search.hotbot.com/hResult.html?SM=SC&MT=%2B%22Martin+Thurn%22+%2BSWB&DE=2&act.next.x=1&base=0&DC=84&
