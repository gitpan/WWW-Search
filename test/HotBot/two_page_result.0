<HTML>
<HEAD><TITLE>HotBot results: +"Martin Thurn" +SWB (1+)</TITLE>
<!-- 'libwww-perl/5.08': ---unknown -->
</HEAD>
<BODY bgcolor="#ffffff" background="/images/bg.base.wide.white.gif" text="#000000" vlink="#ff0000" alink="#0000ff" link="#0000ff"><A HREF="http://help.hotbot.com/?UID=IU1MdaKR5E7CAAB9C9322EF5840101583FF2E52B">Help</A> | <A HREF="http://www.wired.com/">WIRED MAGAZINE</A> | <A HREF="http://www.hotwired.com/">HOTWIRED</A> | <A HREF="http://www.wired.com/wired/">WIRED NEWS</A> | <A HREF="http://www.suck.com/">SUCK.COM</A> | <A HREF="http://www.wired.com/home/">W</A><TABLE cellpadding="0" cellspacing="0">
<TR><TD  valign="CENTER" align="LEFT" NOWRAP><A HREF="http://help.hotbot.com/?UID=IU1MdaKR5E7CAAB9C9322EF5840101583FF2E52B"><IMG SRC="/images/logo.hb4.white.gif"  BORDER=0 WIDTH=195 HEIGHT=45 ALT="- H o t B o t -"></A></TD>
<TD  valign="CENTER" align="RIGHT" NOWRAP width="380"><A HREF="/IU1MdaKR5E7CAAB9C9322EF5840101583FF2E52B/clickthru.html?_id=6673&_h=I0055h1ae&_c=6673&_url=http://turbotax.vitabella.com/%3F1305&_slot=mainCobrand"><IMG SRC="http://images.hotbot.com/ads/turbotax/APRIL1251.GIF"  BORDER=0 WIDTH=125 HEIGHT=35 ALT="Click here for TurboTax"></A></TD></TR>
</TABLE>

<P>
<P>
<FORM action="/IU1MdaKR5E7CAAB9C9322EF5840101583FF2E52B/hResult.html" ><TABLE cellpadding="0" cellspacing="0">
<TR><TD ><IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=130 HEIGHT=1 ALT="&nbsp;"></TD></TR>
<TR ><TD  valign="BOTTOM"></TD>
<TD  valign="BOTTOM" align="RIGHT">
look for&nbsp;<SELECT NAME="SM" >
	<OPTION VALUE="MC">all the words
	<OPTION VALUE="SC" selected>any of the words
	<OPTION VALUE="phrase">the exact phrase
	<OPTION VALUE="title">words in the title
	<OPTION VALUE="name">the person
	<OPTION VALUE="url">links to this URL
	<OPTION VALUE="B">the Boolean phrase
</SELECT></TD></TR>
<TR ><TD >
<FONT FACE="geneva, arial" SIZE=2 COLOR=#FFFFFF><B><I>Search: the Web</I></B></FONT><BR>
</TD>
<TD ><INPUT TYPE=text VALUE="+&quot;Martin Thurn&quot; +SWB" SIZE=39 name="MT" maxlength="300"></TD>
<TD  valign="center"><INPUT TYPE="image" SRC="/images/btn.search-borderless.none.gif" name="act.search" border="0" WIDTH=67 HEIGHT=21 VALUE="Search"></TD></TR>
<TR ><TD ></TD>
<TD  cellspan="2">

</TD></TR>
</TABLE>
<INPUT TYPE="hidden" NAME="RG" VALUE="NA"><INPUT TYPE="hidden" NAME="DC" VALUE="9"><INPUT TYPE="hidden" NAME="_v" VALUE="2"><INPUT TYPE="hidden" NAME="NUMMOD" VALUE="0"></FORM>
<FORM action="http://www.hotbot.com/IU1MdaKR5E7CAAB9C9322EF5840101583FF2E52B/" ><INPUT TYPE="submit" NAME="act.map.newsearch" VALUE="New Search"> | <INPUT TYPE="submit" NAME="act.map.revise" VALUE="Revise Search"><INPUT TYPE="hidden" NAME="MT" VALUE="+&quot;Martin Thurn&quot; +SWB"><INPUT TYPE="hidden" NAME="RG" VALUE="NA"><INPUT TYPE="hidden" NAME="SM" VALUE="SC"><INPUT TYPE="hidden" NAME="DC" VALUE="9"><INPUT TYPE="hidden" NAME="_v" VALUE="2"><INPUT TYPE="hidden" NAME="NUMMOD" VALUE="0"></FORM><TABLE cellpadding="0" cellspacing="0">
<TR><TD ><IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=130 HEIGHT=1 ALT="&nbsp;"></TD>
<TD ><A HREF="/IU1MdaKR5E7CAAB9C9322EF5840101583FF2E52B/clickthru.html?_id=8727&_h=I0055h1ae&_c=8727&_url=http://www.techweb.com/cgi-bin/goto%3FSRC=nbwinanibannerhot%26URL=http://www.cmpnet.com/netbiz&_slot=DEF"><IMG SRC="http://images.hotbot.com/ads/cmpnet/winani.gif"  BORDER=1 WIDTH=468 HEIGHT=60 ALT="click here for e-business solutions: www.techweb.com/netbiz"></A></TD></TR>
<TR ><TD ></TD>
<TD  align="CENTER"><A HREF="/IU1MdaKR5E7CAAB9C9322EF5840101583FF2E52B/clickthru.html?_id=8727&_h=I0055h1ae&_c=8727&_url=http://www.techweb.com/cgi-bin/goto%3FSRC=nbwinanibannerhot%26URL=http://www.cmpnet.com/netbiz&_slot=DEF"><FONT SIZE=2>click here for e-business solutions: www.techweb.com/netbiz</FONT></A></TD></TR>
</TABLE>


<!-- QUERY:
 ( +"Martin Thurn" +SWB)
 -->
<TABLE cellpadding="0" cellspacing="0">
<TR><TD ><IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=136 HEIGHT=1 ALT="&nbsp;"></TD>
<TD >
<B>Returned:</B> 142 matches.
</TD></TR>
</TABLE>
<TABLE cellpadding="0" cellspacing="0">
<TR><TD  valign="TOP" align="RIGHT">
<A HREF="http://www.hotbot.com/">The Web</A> <BR> <A HREF="http://www.hotbot.com/usenet.html">Usenet</A> <BR> <A HREF="http://www.hotbot.com/newsbot/index.html">Top News Sites</A> <BR> <A HREF="http://www.hotbot.com/partners/business.html">Businesses</A> <BR> <A HREF="http://www.hotbot.com/partners/people.html">People</A> <BR> <A HREF="http://www.hotbot.com/partners/email.html">Email Addresses</A> <BR> <A HREF="http://www.classifieds2000.com/cgi-cls/display.exe?hotbot+class">Classifieds</A> <BR> <A HREF="http://www.hotbot.com/partners/websitez.html">Domain Names</A> <BR> <A HREF="http://stocks.hotbot.com/">Stocks</A> <BR> <A HREF="http://www.hotbot.com/partners/forumone.html">Discussion Groups</A> <BR> <A HREF="http://www.hotbot.com/partners/filez.html">ShareWare</A>
<BR>
<IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=115 HEIGHT=1 ALT="&nbsp;"></TD>
<TD ><IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=15 HEIGHT=1 ALT="&nbsp;"></TD>
<TD ><FORM action="/IU1MdaKR5E7CAAB9C9322EF5840101583FF2E52B/hResult.html" ><TABLE cellpadding="0" cellspacing="0">
<TR><TD  width="300" align="LEFT"><B>Breakdown:</B>&nbsp;<FONT size=-1>martin:&nbsp;1736137, thurn:&nbsp;2985, swb:&nbsp;19299</font></TD>
<TD ><IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=17 HEIGHT=1 ALT="&nbsp;"></TD>
<TD >&nbsp;1&nbsp;-&nbsp;9&nbsp;</TD>
<TD ><INPUT TYPE="image" SRC="/images/btn.rightarrow.on.gif" name="act.next" border="0" WIDTH=17 HEIGHT=19 VALUE="Next"></TD></TR>
</TABLE>
<INPUT TYPE="hidden" NAME="MT" VALUE="+&quot;Martin Thurn&quot; +SWB"><INPUT TYPE="hidden" NAME="RG" VALUE="NA"><INPUT TYPE="hidden" NAME="SM" VALUE="SC"><INPUT TYPE="hidden" NAME="DC" VALUE="9"><INPUT TYPE="hidden" NAME="_v" VALUE="2"><INPUT TYPE="hidden" NAME="NUMMOD" VALUE="0"></FORM>
<TABLE cellpadding="0" cellspacing="0">
<TR >
<TD  width="20" align="left" valign="top"><B>1. </B>
</TD>
<TD ><A HREF="http://www.cis.ohio-state.edu/~thurn/SWE/pyramid.html">The Star Wars Collector's Bible (Pyramid Handbags)</A>
</TD></TR>
<TR ><TD  valign="TOP"><FONT size=-1>99%</FONT></TD>
<TD >The Star Wars Collector's Bible is moving in December 1997! The pages located at Ohio State will no longer be updated after Thanksgiving, and are scheduled to be deleted in December. The new URL for this page is http://www.pitt.edu/~thurn/SWB...<BR>
http://www.cis.ohio-state.edu/~thurn/SWE/pyramid.html, 1567 bytes, 21Nov97</TD></TR>
<TR >
<TD  width="20" align="left" valign="top"><B>2. </B>
</TD>
<TD ><A HREF="http://www.cis.ohio-state.edu/~thurn/SWE/pizzahut.html">The Star Wars Collector's Bible (Pizza Hut)</A>
</TD></TR>
<TR ><TD  valign="TOP"><FONT size=-1>99%</FONT></TD>
<TD >The Star Wars Collector's Bible is moving in December 1997! The pages located at Ohio State will no longer be updated after Thanksgiving, and are scheduled to be deleted in December. The new URL for this page is http://www.pitt.edu/~thurn/SWB...<BR>
http://www.cis.ohio-state.edu/~thurn/SWE/pizzahut.html, 1671 bytes, 21Nov97</TD></TR>
<TR >
<TD  width="20" align="left" valign="top"><B>3. </B>
</TD>
<TD ><A HREF="http://www.cis.ohio-state.edu/~thurn/SWE/briefly.html">The Star Wars Collector's Bible (Briefly Stated, Inc.)</A>
</TD></TR>
<TR ><TD  valign="TOP"><FONT size=-1>99%</FONT></TD>
<TD >The Star Wars Collector's Bible is moving in December 1997! The pages located at Ohio State will no longer be updated after Thanksgiving, and are scheduled to be deleted in December. The new URL for this page is http://www.pitt.edu/~thurn/SWB...<BR>
http://www.cis.ohio-state.edu/~thurn/SWE/briefly.html, 1372 bytes, 21Nov97</TD></TR>
<TR >
<TD  width="20" align="left" valign="top"><B>4. </B>
</TD>
<TD ><A HREF="http://www.cis.ohio-state.edu/~thurn/SWE/kidnation.html">The Star Wars Collector's Bible (KidNATION)</A>
</TD></TR>
<TR ><TD  valign="TOP"><FONT size=-1>99%</FONT></TD>
<TD >The Star Wars Collector's Bible is moving in December 1997! The pages located at Ohio State will no longer be updated after Thanksgiving, and are scheduled to be deleted in December. The new URL for this page is http://www.pitt.edu/~thurn/SWB...<BR>
http://www.cis.ohio-state.edu/~thurn/SWE/kidnation.html, 1632 bytes, 21Nov97</TD></TR>
<TR >
<TD  width="20" align="left" valign="top"><B>5. </B>
</TD>
<TD ><A HREF="http://www.cis.ohio-state.edu/~thurn/SWE/hanes.html">The Star Wars Collector's Bible (Hanes)</A>
</TD></TR>
<TR ><TD  valign="TOP"><FONT size=-1>99%</FONT></TD>
<TD >The Star Wars Collector's Bible is moving in December 1997! The pages located at Ohio State will no longer be updated after Thanksgiving, and are scheduled to be deleted in December. The new URL for this page is http://www.pitt.edu/~thurn/SWB...<BR>
http://www.cis.ohio-state.edu/~thurn/SWE/hanes.html, 1593 bytes, 21Nov97</TD></TR>
<TR >
<TD  width="20" align="left" valign="top"><B>6. </B>
</TD>
<TD ><A HREF="http://www.cis.ohio-state.edu/~thurn/SWE/authimages.html">The Star Wars Collector's Bible (Authentic Images)</A>
</TD></TR>
<TR ><TD  valign="TOP"><FONT size=-1>99%</FONT></TD>
<TD >The Star Wars Collector's Bible is moving in December 1997! The pages located at Ohio State will no longer be updated after Thanksgiving, and are scheduled to be deleted in December. The new URL for this page is http://www.pitt.edu/~thurn/SWB...<BR>
http://www.cis.ohio-state.edu/~thurn/SWE/authimages.html, 2093 bytes, 21Nov97</TD></TR>
<TR >
<TD  width="20" align="left" valign="top"><B>7. </B>
</TD>
<TD ><A HREF="http://www.cis.ohio-state.edu/~thurn/SWE/freshcaps.html">The Star Wars Collector's Bible (Fresh Caps Ltd.)</A>
</TD></TR>
<TR ><TD  valign="TOP"><FONT size=-1>99%</FONT></TD>
<TD >The Star Wars Collector's Bible is moving in December 1997! The pages located at Ohio State will no longer be updated after Thanksgiving, and are scheduled to be deleted in December. The new URL for this page is http://www.pitt.edu/~thurn/SWB...<BR>
http://www.cis.ohio-state.edu/~thurn/SWE/freshcaps.html, 1626 bytes, 21Nov97</TD></TR>
<TR >
<TD  width="20" align="left" valign="top"><B>8. </B>
</TD>
<TD ><A HREF="http://www.cis.ohio-state.edu/~thurn/SWE/portal.html">The Star Wars Collector's Bible (Portal Publications)</A>
</TD></TR>
<TR ><TD  valign="TOP"><FONT size=-1>99%</FONT></TD>
<TD >The Star Wars Collector's Bible is moving in December 1997! The pages located at Ohio State will no longer be updated after Thanksgiving, and are scheduled to be deleted in December. The new URL for this page is http://www.pitt.edu/~thurn/SWB...<BR>
http://www.cis.ohio-state.edu/~thurn/SWE/portal.html, 1780 bytes, 21Nov97</TD></TR>
<TR >
<TD  width="20" align="left" valign="top"><B>9. </B>
</TD>
<TD ><A HREF="http://www.cis.ohio-state.edu/~thurn/SWE/milton.html">The Star Wars Collector's Bible (Milton Bradley)</A>
</TD></TR>
<TR ><TD  valign="TOP"><FONT size=-1>99%</FONT></TD>
<TD >The Star Wars Collector's Bible is moving in December 1997! The pages located at Ohio State will no longer be updated after Thanksgiving, and are scheduled to be deleted in December. The new URL for this page is http://www.pitt.edu/~thurn/SWB...<BR>
http://www.cis.ohio-state.edu/~thurn/SWE/milton.html, 1978 bytes, 21Nov97</TD></TR>
</TABLE>


<FORM action="/IU1MdaKR5E7CAAB9C9322EF5840101583FF2E52B/hResult.html" ><TABLE cellpadding="0" cellspacing="0">
<TR><TD  width="300" align="LEFT"><B>Breakdown:</B>&nbsp;<FONT size=-1>martin:&nbsp;1736137, thurn:&nbsp;2985, swb:&nbsp;19299</font></TD>
<TD ><IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=17 HEIGHT=1 ALT="&nbsp;"></TD>
<TD >&nbsp;1&nbsp;-&nbsp;9&nbsp;</TD>
<TD ><INPUT TYPE="image" SRC="/images/btn.rightarrow.on.gif" name="act.next" border="0" WIDTH=17 HEIGHT=19 VALUE="Next"></TD></TR>
</TABLE>
<INPUT TYPE="hidden" NAME="MT" VALUE="+&quot;Martin Thurn&quot; +SWB"><INPUT TYPE="hidden" NAME="RG" VALUE="NA"><INPUT TYPE="hidden" NAME="SM" VALUE="SC"><INPUT TYPE="hidden" NAME="DC" VALUE="9"><INPUT TYPE="hidden" NAME="_v" VALUE="2"><INPUT TYPE="hidden" NAME="NUMMOD" VALUE="0"></FORM>
</TD></TR>
</TABLE>
<TABLE cellpadding="0" cellspacing="0">
<TR><TD ><IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=130 HEIGHT=1 ALT="&nbsp;"></TD>
<TD ><A HREF="/IU1MdaKR5E7CAAB9C9322EF5840101583FF2E52B/clickthru.html?_id=8727&_h=I0055h1ae&_c=8727&_url=http://www.techweb.com/cgi-bin/goto%3FSRC=nbwinanibannerhot%26URL=http://www.cmpnet.com/netbiz&_slot=DEF"><IMG SRC="http://images.hotbot.com/ads/cmpnet/winani.gif"  BORDER=1 WIDTH=468 HEIGHT=60 ALT="click here for e-business solutions: www.techweb.com/netbiz"></A></TD></TR>
<TR ><TD ></TD>
<TD  align="CENTER"><A HREF="/IU1MdaKR5E7CAAB9C9322EF5840101583FF2E52B/clickthru.html?_id=8727&_h=I0055h1ae&_c=8727&_url=http://www.techweb.com/cgi-bin/goto%3FSRC=nbwinanibannerhot%26URL=http://www.cmpnet.com/netbiz&_slot=DEF"><FONT SIZE=2>click here for e-business solutions: www.techweb.com/netbiz</FONT></A></TD></TR>
</TABLE>

<P>
<FORM action="/IU1MdaKR5E7CAAB9C9322EF5840101583FF2E52B/hResult.html" ><TABLE cellpadding="0" cellspacing="0">
<TR><TD ><IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=130 HEIGHT=1 ALT="&nbsp;"></TD></TR>
<TR ><TD  valign="BOTTOM"></TD>
<TD  valign="BOTTOM" align="RIGHT">
look for&nbsp;<SELECT NAME="SM" >
	<OPTION VALUE="MC">all the words
	<OPTION VALUE="SC" selected>any of the words
	<OPTION VALUE="phrase">the exact phrase
	<OPTION VALUE="title">words in the title
	<OPTION VALUE="name">the person
	<OPTION VALUE="url">links to this URL
	<OPTION VALUE="B">the Boolean phrase
</SELECT></TD></TR>
<TR ><TD >
<FONT FACE="geneva, arial" SIZE=2 COLOR=#FFFFFF><B><I>Search: the Web</I></B></FONT><BR>
</TD>
<TD ><INPUT TYPE=text VALUE="+&quot;Martin Thurn&quot; +SWB" SIZE=39 name="MT" maxlength="300"></TD>
<TD  valign="center"><INPUT TYPE="image" SRC="/images/btn.search-borderless.none.gif" name="act.search" border="0" WIDTH=67 HEIGHT=21 VALUE="Search"></TD></TR>
<TR ><TD ></TD>
<TD  cellspan="2">

</TD></TR>
</TABLE>
<INPUT TYPE="hidden" NAME="RG" VALUE="NA"><INPUT TYPE="hidden" NAME="DC" VALUE="9"><INPUT TYPE="hidden" NAME="_v" VALUE="2"><INPUT TYPE="hidden" NAME="NUMMOD" VALUE="0"></FORM>
<FORM action="http://www.hotbot.com/IU1MdaKR5E7CAAB9C9322EF5840101583FF2E52B/" ><INPUT TYPE="submit" NAME="act.map.newsearch" VALUE="New Search"> | <INPUT TYPE="submit" NAME="act.map.revise" VALUE="Revise Search"><INPUT TYPE="hidden" NAME="MT" VALUE="+&quot;Martin Thurn&quot; +SWB"><INPUT TYPE="hidden" NAME="RG" VALUE="NA"><INPUT TYPE="hidden" NAME="SM" VALUE="SC"><INPUT TYPE="hidden" NAME="DC" VALUE="9"><INPUT TYPE="hidden" NAME="_v" VALUE="2"><INPUT TYPE="hidden" NAME="NUMMOD" VALUE="0"></FORM>

<A HREF="http://www.inktomi.com/"><IMG SRC="/images/logo.inktomi.white.gif"  BORDER=0 WIDTH=126 HEIGHT=40 ALT="powered by INKTOMI"></A><TABLE cellpadding="0" cellspacing="0">
<TR><TD ><IMG SRC="/images/misc.transPixel.gif"  BORDER=0 WIDTH=130 HEIGHT=1 ALT="&nbsp;"></TD>
<TD ><FONT face="Arial, Helvetica" size=1>
	Copyright &#169; 1996-97 Wired Digital, Inc. All rights reserved.
    </FONT></TD></TR>
</TABLE>
</BODY>
</HTML>
<!-- time = 248291 microseconds per iteration -->


