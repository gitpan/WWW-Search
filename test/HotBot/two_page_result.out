http://www.cis.ohio-state.edu/~thurn/SWE/pyramid.html
http://www.cis.ohio-state.edu/~thurn/SWE/pizzahut.html
http://www.cis.ohio-state.edu/~thurn/SWE/briefly.html
http://www.cis.ohio-state.edu/~thurn/SWE/kidnation.html
http://www.cis.ohio-state.edu/~thurn/SWE/hanes.html
http://www.cis.ohio-state.edu/~thurn/SWE/authimages.html
http://www.cis.ohio-state.edu/~thurn/SWE/freshcaps.html
http://www.cis.ohio-state.edu/~thurn/SWE/portal.html
http://www.cis.ohio-state.edu/~thurn/SWE/milton.html
http://www.cis.ohio-state.edu/~thurn/SWE/ahprismatic.html
http://www.cis.ohio-state.edu/~thurn/SWE/freeze.html
http://www.cis.ohio-state.edu/~thurn/SWE/eddies.html
http://www.cis.ohio-state.edu/~thurn/SWE/decipher.html
http://www.cis.ohio-state.edu/~thurn/SWE/icons.html
http://www.cis.ohio-state.edu/~thurn/SWE/tinsigns.html
http://www.cis.ohio-state.edu/~thurn/SWE/polydata.html
http://www.cis.ohio-state.edu/~thurn/SWE/pca.html
http://www.cis.ohio-state.edu/~thurn/SWE/littlebrown.html
http://www.cis.ohio-state.edu/~thurn/SWE/goldturt.html
http://www.cis.ohio-state.edu/~thurn/SWB/portal.html
http://www.cis.ohio-state.edu/~thurn/SWB/briefly.html
http://www.cis.ohio-state.edu/~thurn/SWB/kidnation.html
http://www.cis.ohio-state.edu/~thurn/SWB/freshcaps.html
http://www.cis.ohio-state.edu/~thurn/SWB/authimages.html
http://www.cis.ohio-state.edu/~thurn/SWB/freeze.html
http://www.cis.ohio-state.edu/~thurn/SWB/topps-heads.html
http://www.cis.ohio-state.edu/~thurn/SWB/topps-master.html
http://www.cis.ohio-state.edu/~thurn/SWB/decipher.html
http://www.cis.ohio-state.edu/~thurn/SWB/topps-ESBlarge.html
http://www.cis.ohio-state.edu/~thurn/SWB/topps-vehicles.html
http://www.cis.ohio-state.edu/~thurn/SWE/abbrevs.html
http://www.cis.ohio-state.edu/~thurn/SWE/kenner.html
http://www.cis.ohio-state.edu/~thurn/SWE/rubies.html
http://www.cis.ohio-state.edu/~thurn/SWE/highbridge.html
http://www.cis.ohio-state.edu/~thurn/SWB/topps-caps.html
http://www.cis.ohio-state.edu/~thurn/SWB/highbridge.html
http://www.cis.ohio-state.edu/~thurn/SWE/rolling.html
http://www.pitt.edu/~thurn/SWB/freeze.html
http://www.pitt.edu/~thurn/SWB/littlebrown.html
http://www.pitt.edu/~thurn/SWB/kenner.html
http://www.pitt.edu/~thurn/SWB/advanced.html
http://www.pitt.edu/~thurn/SWB/abbrevs.html
http://www.pitt.edu/~thurn/SWB/briefly.html
http://www.pitt.edu/~thurn/SWB/topps-heads.html
http://www.cis.ohio-state.edu/~thurn/SWE/antioch.html
http://www.cis.ohio-state.edu/~thurn/SWE/hope.html
http://www.pitt.edu/~thurn/SWB/topps-pinups.html
http://www.cis.ohio-state.edu/~thurn/SWB/ahprismatic.html
http://www.pitt.edu/~thurn/SWB/ahprismatic.html
http://www.pitt.edu/~thurn/SWB/antioch.html
http://www.pitt.edu/~thurn/SWB/decipher.html
http://www.cis.ohio-state.edu/~thurn/SWB/advanced.html
http://www.cis.ohio-state.edu/~thurn/SWE/microgames.html
http://www.cis.ohio-state.edu/~thurn/SWE/hamilton.html
http://www.pitt.edu/~thurn/SWB/kidnation.html
http://www.pitt.edu/~thurn/SWB/microgames.html
http://www.cis.ohio-state.edu/~thurn/SWE/mead.html
http://www.cis.ohio-state.edu/~thurn/SWB/microgames.html
http://www.cis.ohio-state.edu/~thurn/SWB/topps-pinups.html
http://www.cis.ohio-state.edu/~thurn/SWE/berkley.html
http://www.cis.ohio-state.edu/~thurn/SWE/donpost.html
http://www.cis.ohio-state.edu/~thurn/SWB/berkley.html
http://www.cis.ohio-state.edu/~thurn/SWE/parker.html
http://www.cis.ohio-state.edu/~thurn/SWE/zanart.html
http://www.cis.ohio-state.edu/~thurn/SWB/parker.html
http://www.cis.ohio-state.edu/~thurn/SWB/rolling.html
http://www.cis.ohio-state.edu/~thurn/SWE/willitts.html
http://www.cis.ohio-state.edu/~thurn/SWE/lookup.html
http://www.cis.ohio-state.edu/~thurn/SWE/metallic.html
http://www.cis.ohio-state.edu/~thurn/SWE/amt.html
http://www.cis.ohio-state.edu/~thurn/SWB/willitts.html
http://www.cis.ohio-state.edu/~thurn/SWB/metallic.html
http://www.pitt.edu/~thurn/SWB/lookup.html
http://www.pitt.edu/~thurn/SWB/metallic.html
http://www.cis.ohio-state.edu/~thurn/SWB/topps-finest.html
http://www.cis.ohio-state.edu/~thurn/SWE/rawcliffe.html
http://www.cis.ohio-state.edu/~thurn/SWE/tiger.html
http://www.cis.ohio-state.edu/~thurn/SWE/roseart.html
http://www.pitt.edu/~thurn/SWB/amt.html
http://www.cis.ohio-state.edu/~thurn/SWB/rawcliffe.html
http://www.cis.ohio-state.edu/~thurn/SWE/tacobell.html
http://www.cis.ohio-state.edu/~thurn/SWE/opeechee.html
http://www.cis.ohio-state.edu/~thurn/SWB/dh-de.html
http://www.cis.ohio-state.edu/~thurn/SWE/hollywood.html
http://www.cis.ohio-state.edu/~thurn/SWB/roseart.html
http://www.cis.ohio-state.edu/~thurn/SWE/changes.html
http://www.cis.ohio-state.edu/~thurn/SWE/applause.html
http://www.pitt.edu/~thurn/SWB/changes.html
http://www.cis.ohio-state.edu/~thurn/SWE/random.html
http://www.cis.ohio-state.edu/~thurn/SWB/art-darkhorse.html
http://www.cis.ohio-state.edu/~thurn/SWB/applause.html
http://www.pitt.edu/~thurn/SWB/applause.html
http://www.cis.ohio-state.edu/~thurn/SWB/random.html
http://www.pitt.edu/~thurn/SWB/t.html
http://www.pitt.edu/~thurn/SWB/index.html
http://www.cis.ohio-state.edu/~thurn/SWE/justtoys.html
http://www.cis.ohio-state.edu/~thurn/SWE/t.html
http://www.cis.ohio-state.edu/~thurn/SWE/
http://www.cis.ohio-state.edu/~thurn/SWB/topps-SE.html
http://www.pitt.edu/~thurn/SWB/justtoys.html
http://www.pitt.edu/~thurn/SWB/ballantine.html
http://www.pitt.edu/~thurn/SWB/m.html
http://www.cis.ohio-state.edu/~thurn/SWB/topps-ROJ.html
http://www.cis.ohio-state.edu/~thurn/SWE/marvel.html
http://www.cis.ohio-state.edu/~thurn/SWB/ballantine.html
http://www.pitt.edu/~thurn/SWB/marvel.html
http://www.pitt.edu/~thurn/SWB/topps-galaxy.html
http://www.cis.ohio-state.edu/~thurn/SWB/topps-galaxy.html
http://www.cis.ohio-state.edu/~thurn/SWB/companies.html
http://www.pitt.edu/~thurn/SWB/companies.html
http://www.cis.ohio-state.edu/~thurn/SWE/companies.html
http://www.cis.ohio-state.edu/~thurn/SWE/darkhorse.html
http://www.cis.ohio-state.edu/~thurn/SWB/darkhorse.html
http://www.cis.ohio-state.edu/~thurn/SWB/topps-ESB.html
http://www.pitt.edu/~thurn/SWB/topps-ESB.html
http://www.pitt.edu/~thurn/SWB/magazines.html
http://www.cis.ohio-state.edu/~thurn/SWB/calendar.html
http://www.pitt.edu/~thurn/SWB/calendar.html
http://www.cis.ohio-state.edu/~thurn/SWB/topps-SW.html
http://www.cis.ohio-state.edu/~thurn/SWE/calendar.html
http://www.cis.ohio-state.edu/~thurn/SWB/tpromos.html
http://www.cis.ohio-state.edu/~thurn/SWE/tpromos.html
http://www.cis.ohio-state.edu/~thurn/SWE/westend.html
http://www.pitt.edu/~thurn/SWB/tpromos.html
http://www.cis.ohio-state.edu/~thurn/SWE/galoob.html
http://www.cis.ohio-state.edu/~thurn/SWB/galoob.html
http://www.cis.ohio-state.edu/~thurn/SWB/westend.html
http://www.phoaks.com/phoaks/rec/arts/sf/starwars/collecting/resources0.html
http://stwing.resnet.upenn.edu/starwars/collectfaq.txt
http://swa.simplenet.com/iforce/news/collecting/collectf.txt
http://empire.res.wabash.edu/text/collectf.txt
http://www.stwing.upenn.edu/starwars/collectfaq.txt
http://www.cs.washington.edu/homes/lopez/faq.html
http://www.themall.net/~diamond2/swcolfaq.txt
http://www.xmission.com:8000/~gastown/cfti/sw-faq.txt
http://198.60.22.4/~gastown/cfti/sw-faq.txt
http://www1.tripnet.se/~topcat/starwars/ovrigt/swcolfaq.txt
http://pages.map.com/starwars/faq.txt
